content/doc/book/managing/system-properties.adoc

```diff
@@ -2060,7 +2060,37 @@ properties:
   def: 20 seconds
   since: workflow-durable-task-step-plugin 2.29
   description: |
-    How long to wait for remote calls (see https://issues.jenkins.io/browse/JENKINS-46507[JENKINS-46507]).
+    How long to wait, in seconds, before interrupting remote calls and forcing cleanup when the step is stopped.
+    See https://issues.jenkins.io/browse/JENKINS-46507[JENKINS-46507] for more information.
+
+- name: org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep.USE_WATCHING
+  tags:
+  - feature
+  def: |
+    `false`
+  since: workflow-durable-task-step-plugin 2.22
+  description: |
+    `true` to enable the experimental push mode for durable task logging.
+    See https://issues.jenkins.io/browse/JENKINS-52165[JENKINS-52165] for more information.
+
+- name: org.jenkinsci.plugins.workflow.support.pickles.ExecutorPickle.timeoutForNodeMillis
```

Comment:
Note: moved around in https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/180 but the name kept for compatibility.